ElasticBox/reporting.md

```diff
@@ -13,9 +13,21 @@ Reports show cloud costs and resource usage in public and private clouds. It inc
 
 If reporting is turned on for your enterprise, ElasticBox Administrators can access it from the admin console. Reports also help you make budget-friendly decisions and curb costs through cost center budgeting.
 
+<<<<<<< HEAD
```

Comment:
Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.